.pylintrc

```diff
@@ -0,0 +1,211 @@
+[MASTER]
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS,migrations
+
+# Pickle collected data for later comparisons.
+persistent=yes
+
+# List of plugins (as comma separated values of python modules names) to load
+load-plugins=pylint_django
+
+[MESSAGES CONTROL]
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time.
+disable=W0403,W0232,E1101,C0111,I0011
```

Comment:
Can we leave here only `disable=E1101,I0011`? Was that deliberated decision to point here also `W0403`, `W0232` and `C0111`?